site/app/models/GradeableTestcase.php

```diff
@@ -152,7 +152,7 @@ public function isExtraCredit() {
     }
     
     public function hasDetails() {
-      return (!$this->isHidden() && count($this->autochecks) > 0);
+      return (count($this->autochecks) > 0);
```

Comment:
while you're modifying this model, can you add the `/** @property @var bool */` comment to both `$hidden` and `$extra_credit`, and then we can remove the `isHidden()` and `isExtraCredit()` functions (add them to the block of `@method ...` above the class name.